assets/js/components/KeyMetrics/hooks/useDisplayCTAWidget.js

```diff
@@ -46,6 +47,10 @@ export default function useDisplayCTAWidget() {
 		return select( MODULES_SEARCH_CONSOLE ).isDataAvailableOnLoad();
 	} );
 	const analyticsIsDataAvailableOnLoad = useSelect( ( select ) => {
+		if ( ! select( CORE_MODULES ).isModuleConnected( 'analytics-4' ) ) {
+			return false;
+		}
+
 		select( MODULES_ANALYTICS_4 ).isGatheringData();
```

Comment:
This looks like it fixes the issue, however, `isGatheringData` seems to be triggered to invoke the side-effect of requesting data to test which is only done if data is not available on load. This shouldn't be necessary and won't change the result anyways since the `isDataAvailableOnLoad` selector only returns the value that was present on-load. We can keep the added guard as it's conceptually correct, but the main problem is the call to `isGatheringData`.